customer.go

```diff
@@ -0,0 +1,31 @@
+package xendit
+
+type CustomerAddress struct {
```

Comment:
Can add a comment for Documentation in pkg.go.dev?

Reply:
I'm actually kinda confused about this. There is that kind of comment in other examples but I can not find `Customer` in pkg.go.dev. What should I do? Am I understanding it wrong?

Reply:
I think the comment will be reflected once we release the SDK version

Reply:
You can try to run `godoc` in your local to see if it's works. 
If you can see the docs in local by using `godoc` you should be able to see it later in pkg.go.dev  once we release new version

Reply:
I try to install `godoc` by typing this command `go get -v  golang.org/x/tools/cmd/godoc`. But after that when I try to run `godoc` it's said that command not found. Am I doing it wrong? @bxcodec

Reply:
Ah nvm, is this what you mean? @bxcodec 
![image](https://user-images.githubusercontent.com/60346343/127591250-939a991c-ea8f-4aa1-bf20-da16262c0843.png)

Reply:
I've installed `godoc` and when I run `godoc -http=:6060`, it redirects me into this page but I can't find the documentation for the project.
![image](https://user-images.githubusercontent.com/60346343/127595273-0f2406dd-1f04-448a-9b6c-06420c7b34a6.png)
I've also tried to change the `pkg.go.dev` here https://pkg.go.dev/github.com/xendit/xendit-go into `localhost:6060` (it becomes `localhost:6060/github.com/xendit/xendit-go`) but it didn't work as well. Am I doing it wrong? @bxcodec

Reply:
The issue has been resolved